integration-tests/fixtures/generate-bundle-stats/webpack/webpack-plugin.test.ts

```diff
@@ -1,37 +1,55 @@
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
 import { $ } from "bun";
-import { describe, it, expect, afterEach } from "bun:test";
+import { describe, it, expect, afterEach, beforeEach } from "bun:test";
 
 const webpackPath = (version: number) =>
   `node_modules/webpackV${version}/bin/webpack.js`;
-const webpackConfig = (version: number) =>
-  `fixtures/generate-bundle-stats/webpack/webpack-v${version}.config.cjs`;
-const webpackApp = "test-apps/vite";
+const webpackConfig = (version: number, format: string) =>
+  `fixtures/generate-bundle-stats/webpack/webpack-v${version}-${format}.config.cjs`;
+const webpackApp = "test-apps/webpack";
+const genConfigScript = "scripts/gen-config.sh";
+
+const VERSIONS = [5];
+const FORMATS = [
+  { format: "array-push", expected: "array-push" },
+  { format: "commonjs", expected: "cjs" },
+  { format: "module", expected: "esm" },
```

Comment:
Why only these formats for webpack?

Reply:
https://webpack.js.org/configuration/output/#outputchunkformat

Reply:
These are the main ones supported out of the box by webpack, tho you can further config with plugins

Reply:
Let's leave a comment with links to those docs!

Reply:
Done, and added some in for Rollup and Vite, as well